extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -601,6 +609,30 @@ class AmpAccordion extends AMP.BaseElement {
       tryFocus(newFocusHeader);
     }
   }
+
+  /**
+   * Callback function to execute when mutations are observed.
+   * @param {!Array<!MutationRecord>} mutations
+   */
+  toggleMutations_(mutations) {
+    if (mutations.length > 1) {
```

Comment:
Why does this check exist? If expanded changed at the same time as some other attribute, why would you ignore the change? What if a developer binds `expanded` and `aria-label`?

Reply:
Can developers bind on `aria-label` attributes? I'm not seeing anything about it in the [supported attributes](https://amp.dev/documentation/components/amp-bind?referrer=ampproject.org#bindings). This won't be a problem any more after including `attributeFilter` in the options anyway, right?

Reply:
We should update that documentation. Here are the allowed global bindings:

https://github.com/ampproject/amphtml/blob/7007e47938b5a1f92191accc2c8c4ab81647a97b/validator/validator-main.protoascii#L6599-L6636